### PR TITLE
Initial structure

### PR DESCRIPTION
**Pull request by gersongcg**
# Corporate Theme - Initial Phase

This theme is contains customised and new theme templates only. 

Preview theme: https://partner-webinar.storeconnectdemo.com/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,61 @@
...
-StoreConnect Corporate Theme
+# Corporate Theme
+
+This is a multi-purpose theme, ideal for corporate, B2B, or services companies.
+
+## Directory Structure
+
+This theme has a specific folder structure. Some folders may be empty, but they need to remain part of the theme structure as they are not overridden or used for new templates.
+
+Also, it has been developed using the [Base Theme](https://github.com/GetStoreConnect/base-theme) as a boilerplate. The Corporate theme doesn't contain all the base theme structure, instead, we only include those customised, modified and new templates, snippets, variables or translations. Feel free to create a brand new theme using the base theme.
```
Comment (toddweetman) [inline README.md]:
```suggestion
Also, it has been developed using the [Base Theme](https://github.com/GetStoreConnect/base-theme) as a boilerplate. The Corporate theme doesn't contain all the base theme structure, instead, we only include those customised, modified and new templates, snippets, variables or translations. Feel free to create a brand new theme using the base theme or this one.
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,61 @@
-# corporate-theme
-StoreConnect Corporate Theme
+# Corporate Theme
+
+This is a multi-purpose theme, ideal for corporate, B2B, or services companies.
+
+## Directory Structure
+
+This theme has a specific folder structure. Some folders may be empty, but they need to remain part of the theme structure as they are not overridden or used for new templates.
```
Comment (toddweetman) [inline README.md]: I don't understand this:
> as they are not overridden or used for new templates.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,61 @@
...
+
+Also, it has been developed using the [Base Theme](https://github.com/GetStoreConnect/base-theme) as a boilerplate. The Corporate theme doesn't contain all the base theme structure, instead, we only include those customised, modified and new templates, snippets, variables or translations. Feel free to create a brand new theme using the base theme.
+
+## Theme Installer
+
+The theme installer is in your Salesforce org, where you must upload this theme as a zip file. Before uploading, remove the .git directory and README.md file from this repo. The theme installer displays errors if it does not find the standard directory structure.
```
Comment (toddweetman) [inline README.md:13]: Don't think you need to worry about the extra files anymore

File: README.md [hunk 4]
```diff
@@ -1,2 +1,61 @@
...
+- featured_categories
+- featured_products
+
+## How to Add Content Blocks to Picklist
+
+You can follow the instructions in this document to [add content blocks to the picklist](https://help.getstoreconnect.com/documentation/adding-templates-to-content-template-picklist.html).
+
+## Configuration
+
+- Ensure that your root directory does not contain any `.git` hidden folders or files
```
Comment (toddweetman) [inline README.md:47]: check this but don't think this step is necessary. Seems only requirement is that root folder is named `theme`

**Review by toddweetman — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (gersongcg) [reply README.md:13]: @toddweetman I'm not fully confident to say that you can upload any type of files as I haven't tested various situations yet.

File: README.md (hunk 4 above)
Comment (gersongcg) [reply README.md:47]: @toddweetman we should keep it as it if for the same reason mentioned above.

**Review by toddweetman — APPROVED** [review]
We can approve this to get it published. There will be needed fixes, no doubt and other improvements. For now this is a workable theme.

**Pull request merged**